docs/troubleshooting/authentication/code_403_the_caller_does_not_have_permission_status_permission_denied.md

```diff
@@ -0,0 +1,35 @@
+---
+keywords: ['permission', 'permission_denied', 'code']
+slug: /code-403-the-caller-does-not-have-permission-status-permission-denied
+title: 'Code: 403, The caller does not have permission. status: PERMISSION_DENIED'
+---
+# Code: 403, The caller does not have permission. status: PERMISSION_DENIED
+
+This error typically occurs when your application or service account does not have the required permissions to access a resource in Google Cloud or Firebase.
+
+## Code 403 Error Message
+
+You may encounter this error due to one or more of the following reasons:
+
+- **Invalid or misconfigured service account JSON file**
+- **Insufficient permissions** assigned to the service account
+- **Missing or incorrect IAM roles** for the service account
+- **API not enabled** in the Google Cloud project
+
+
+## How to Fix:
+
+### 1. Check your service account JSON file
```

Comment:
This is inconsistent to other files. Have you seen anywhere where we use numbered bullets as headers? I don't think I mentioned it anywhere else too.